scripts/fetch-asyncapi-example.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const request = require('request');
+const fs = require('fs');
+const unzipper = require('unzipper');
+const path = require('path');
+const parser = require('@asyncapi/parser');
+
+const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/heads/master.zip';
+const EXAMPLE_DIRECTORY = path.join(__dirname, '../assets/examples');
+const TEMP_ZIP_NAME = 'spec-examples.zip';
+
+const fetchAsyncAPIExamplesFromExternalURL = () => {
+  return new Promise((resolve, reject) => {
+    request(SPEC_EXAMPLES_ZIP_URL)
+      .pipe(fs.createWriteStream(TEMP_ZIP_NAME))
+      .on('close', () => {
+        console.log('Fetched ZIP file');
+        resolve();
+      }).on('error', () => {
+        reject();
+      });
```

Comment:
Optional suggestion 😄 

```suggestion
      }).on('error', reject);
```